Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -536,7 +536,7 @@ CharacterRange AXTextMarker::characterRangeForLine(unsigned lineIndex) const
     // This implementation doesn't respect the offset as the only known callsite hardcodes zero. We'll need to make changes to support this if a usecase arrives for it.
     RELEASE_ASSERT(!offset());
 
-    auto* stopObject = object->nextUnignoredSiblingOrParent();
+    auto* stopObject = object->nextUnignoredSiblingOrParent(true);
```

Comment:
Thank you for fixing my mistake! I wonder if we should just remove the parameter outright, since no existing caller needs to pass a `false` 🤔